src/NUnitFramework/tests/Attributes/RangeAttributeTests.cs

```diff
@@ -408,7 +412,11 @@ private void CheckValues(string methodName, params object[] expected)
         {
             var method = GetType().GetMethod(methodName, BindingFlags.NonPublic | BindingFlags.Instance);
             var param = method.GetParameters()[0];
+#if NETCORE
+            var attr = param.GetCustomAttributes(typeof(ValuesAttribute), false).First() as ValuesAttribute;
```

Comment:
GetCustomAttributes returns IEnumerable on DNX. This was one of the few places that caused a problem.